styles/home.module.scss

```diff
@@ -0,0 +1,50 @@
+@use 'variables.module' as *;
+
+.container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  overflow-x: auto;
+  -webkit-overflow-scrolling: touch;
+  img { margin-right: 15px; }
+
+  align-items: flex-start;
+  height: 75vh;
+}
+
+.create {
+  margin-left: 17.5vh;
+}
+
+.single {
+  margin: auto;
+}
+
+.offset {
+  margin-left: 35vh;
+  margin-top: -4.5vh;
+}
+
+.top:nth-child(odd) {
+  margin-left: -17.5vh;
+}
+
+.top:nth-child(even) {
+  margin-top: -4.5vh;
+}
```

Comment:
i can only imagine this isn't very mobile friendly..

Reply:
i think we're planning to do a very different mobile design in the future? would probably have to revamp most of the code for mobile so this part of the code could just be desktop-only